src/app/app.component.ts

```diff
@@ -1,10 +1,25 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import content from '../assets/content.json';
+import { ArticleFormat } from './format.js';
+import { ArticleListService } from './article-list.service.js';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'NewsFeed';
-}
+export class AppComponent implements OnInit{
+
+  static refrenceObject : ArticleListService;
```

Comment:
I would recommend doing the below changes

- Instead of having a static reference on the component level, you could have used a service for having a reference for this variable. Inject that service in those components where it is required and get the data.
- Services are meant for that purpose
- Components are only for the presentation of the view

Reply:
Replaced the use of `static `reference member of component to access the Data by using `service `instead